src/cli.rs

```diff
@@ -9,6 +9,31 @@ use url::Url;
 const COPYRIGHT: &str = "Copyright © 2021 by conplement AG";
 
 // ToDo: command completion
+#[derive(Parser, Debug)]
+#[command(after_help = COPYRIGHT)]
+/// inject and manage docker containers in a firmware image
+pub enum Docker {
+    Inject {
+        /// full qualified name of the docker image
+        #[clap(short = 'd', long = "docker-image", required(true))]
+        docker_image: String,
+        /// path to wic image file (optionally compressed with xz, bzip2 or gzip)
+        #[arg(short = 'i', long = "image")]
+        image: PathBuf,
+        /// partition to store the image to
+        #[clap(short = 'a', long = "partition", value_enum)]
+        partition: Partition,
+        /// file path to store the docker image to
+        #[clap(short = 'e', long = "dest")]
+        dest: PathBuf,
```

Comment:
wouldn't it make sense to set a default here as well?

Reply:
Is there a sensible default? @JoergZeidler ?

Reply:
default could be the root path of the partition

Reply:
imho a better default would be 
factory partition and "/oci_images" as default path.